<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventDialog</class>
 <widget class="QDialog" name="AddEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sports Scoring System - Add Event</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Yeargroups: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Event: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Sections: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/data/list-remove.png</normaloff>:/data/list-remove.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/data/list-add.png</normaloff>:/data/list-add.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/data/go-up.png</normaloff>:/data/go-up.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDown">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/data/go-down.png</normaloff>:/data/go-down.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	font: italic;
}</string>
   </property>
   <property name="text">
    <string>Type has to be either TIME or TEXT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEventDialog</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok()</slot>
 </slots>
</ui>
